main.js

```diff
@@ -152,9 +152,15 @@ const callPolice = async () => {
     resDataSort.push(resData[key].category)
   }
   // reduce the array to key value pairs of category and occurences
-  const occurrences = resDataSort.reduce(function (acc, curr) {
-    return acc[curr] ? ++acc[curr] : acc[curr] = 1, acc
-  }, {});
+  let occurrences = {};
+  for (let i = 0; i < resDataSort.length; i++) {
+    let element = resDataSort[i];
+    if (occurrences[element]) {
```

Comment:
This is probably annoying, but I played around with the previous version on runjs and got to understanding it.

I think you actually got to the better syntax for this because it was straightforward and avoided a loop by just using .reduce(), which is great - we just didn't understand it because we're noobs 🤣 

but now we're learning and that was a nice piece of code, so I think we should leave it alone